tools/custom/raw-bytecode-analyser/raw_bytecode_analyser.py

```diff
@@ -0,0 +1,55 @@
+from evmdasm import EvmBytecode
+import argparse
+import requests
+
+def fetch_contract_bytecode(mirror_node_url, contract_id):
+    url = f'{mirror_node_url}/api/v1/contracts/{contract_id}'
+    response = requests.get(url)
+
+    if response.status_code == 200:
+        bytecode = response.json().get('runtime_bytecode')
+        if bytecode:
+            address_detected = False
+            method_detected = False
+            param_detected = False
+            call_detected = False
+
+            evm_bytecode = EvmBytecode(bytecode)
+            disassembled_code = evm_bytecode.disassemble()
```

Comment:
Is it worth to include a more precise analysis? Slither might have a mechanism to include this kind of detection that relies on the CFG of the contract rather than the disassembly.

Reply:
@acuarica yes, but:
 1. Slither requires AST to work correctly: https://github.com/arianejasuwienas/hedera-smart-contracts/pull/2/files#diff-a0145f8b7b5f0359d5db591110cc968501542b54507461c7ccf21ffc6ee53367
 2. Slither detector is part of different PR: https://github.com/arianejasuwienas/hedera-smart-contracts/pull/1/files